integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -118,64 +105,52 @@ spec:
             name: rpc
           - containerPort: 19999
             name: web
-          {{- if $isEmbedded }}
           - containerPort: 19200
             name: embedded
-          {{- end}}
           volumeMounts:
             {{- if $needJournalVolume }}
             - name: alluxio-journal
               mountPath: /journal
             {{- end }}
-            {{- if $root.Values.master.metastore }}
+            {{- if .Values.master.metastore }}
             - name: alluxio-metastore
-              mountPath: {{ $root.Values.master.metastore.mountPath }}
+              mountPath: {{ .Values.master.metastore.mountPath }}
             {{- end }}
-            {{- if $root.Values.secrets }}
-              {{- if $root.Values.secrets.master }}
-                {{- range $key, $val := $root.Values.secrets.master }}
-            - name: secret-{{ $key }}-volume
-              mountPath: /secrets/{{ $val }}
-              readOnly: true
-                {{- end }}
+            {{- if .Values.secrets }}
+              {{- if .Values.secrets.master }}
+{{- include "alluxio.master.secretVolumeMounts" . | indent 10 }}
```

Comment:
Secret volumes are shared by master and job-master. Moved this dup block into template.